src/components/BlockCapabilityBlockDocumentInput.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="block-capability-block-document-input" :class="classes" :style="styles" v-bind="listeners">
-    <BlockDocumentsSelect v-model:selected="internalModalValue" class="block-capability-block-document-input__select" :block-documents="blockDocuments" v-bind="attrs" required>
```

Comment:
Is there a reason not to update the underlying component? just curious

Reply:
BlockDocumentsSelect is more generic. And currently its only used with a single block type. Technically it could be used with multiple block types. But didn't seem like the complexity was necessary. We can always add it later if we want.

Reply:
fair enough 🤷🏻